api/routes/device_test.go

```diff
@@ -14,11 +18,66 @@ import (
 	"github.com/shellhub-io/shellhub/api/pkg/guard"
 	svc "github.com/shellhub-io/shellhub/api/services"
 	"github.com/shellhub-io/shellhub/api/services/mocks"
+	"github.com/shellhub-io/shellhub/pkg/api/paginator"
 	"github.com/shellhub-io/shellhub/pkg/errors"
 	"github.com/shellhub-io/shellhub/pkg/models"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestGetDevice(t *testing.T) {
+	e := echo.New()
+	e.Validator = handlers.NewValidator()
+	mock := new(mocks.Service)
+	ctx := context.TODO()
+	h := NewHandler(mock)
+
+	cases := []struct {
+		title         string
+		uid           string
+		requiredMocks func()
+		expectedErr   error
+	}{
+		{
+			title: "Get device success",
+			uid: `{
+				"uid": "123"
+			}`,
```

Comment:
```suggestion
			uid: `{ "uid": "123" }`,
```

Reply:
The same code style could be applied in all tests as it reduces code size.

Reply:
resolved during refactoring